books/canute.ncbi news summer 2015 g2.pef.txt

```diff
@@ -1,2 +0,0 @@
-current_page = 1
-bookmarks = []
```

Comment:
Is there a reason you are committing this state file? Should we perhaps make an addition to the .gitignore?